integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -48,4 +71,30 @@ export class RasaNlu extends NluPlugin<RasaNluConfig> {
   ): Promise<AxiosResponse<RasaResponse>> {
     return axios.post(`${this.config.serverUrl}${this.config.serverPath}`, { text }, config);
   }
+
+  private mapAlternativeIntents(allIntents: RasaIntent[]): RasaIntent[] {
+    // remove first element, because its the classified intent
+    const alternativeIntents = allIntents.slice(1);
+
+    return alternativeIntents
+      .filter((a) => a.confidence > this.config.alternativeIntents.confidenceCutoff)
+      .slice(0, this.config.alternativeIntents.maxAlternatives);
+  }
+
+  private static mapEntities(entityMap: EntityMap, rasaEntity: RasaEntity): EntityMap {
```

Comment:
Not sure if this has to be static, theoretically an instance-method should suffice.

Reply:
i thought it is good practice to make methods static that do not access any instance variables. What do you think?

Reply:
I've read that too somewhere but I am not a huge fan of that personally. Being able to access the method via `this` seems cleaner to me and there should not be a noticeable difference in terms of performance either.

 I guess going with the instance method would be better just for the sake of fitting the code style of the other packages.

Reply:
Ok that's fine with me. Done.